src/routes/search/containers/SearchPage.js

```diff
@@ -32,15 +32,27 @@ export class SearchPage extends React.Component {
   }
```

Comment:
The sorting should be tested (A few examples for sorting should be added in the spec)

Reply:
done